src/external_connections/next_gen_io.py

```diff
@@ -0,0 +1,61 @@
+#Utility I/O functions for the next generation water model network.
+#The read_qlat and replace_downstreams functions are copies from nhd_io.py
+#and are placed here for now, so that this program can run without importing that module.
+import xarray as xr
+import pandas as pd
+import geopandas as gpd
+import os
+
+
+def read_ngen_network_csv(path):
+    return pd.read_csv(path, index_col=0)
+
+
+def read_ngen_network_geojson(path):
+    return gpd.read_file(path)
+
+
+def read_ngen_network_json(path):
+    return pd.read_json(path)
+
+
+#Convert catchment lateral flows to format that can be processed by compute_network
+def read_catchment_lateral_flows(path):
+    ql = []
+    qlats_df = pd.DataFrame({'A' : []})
+
+    past_first_loop = False
+
+    catchment_id_list = []
+
+    for file_name in os.listdir(path):
+       if file_name.startswith("cat-"):
+          file_name_str_list = file_name.split("_")
+          catchment_id = int(file_name_str_list[0][4 :])
+          catchment_id_list.append(catchment_id)
+          catchment_qlats = pd.read_csv(file_name, names=["", catchment_id])
+          catchment_qlats = catchment_qlats.iloc[:, [1]]
+          catchment_qlats_transposed = catchment_qlats.transpose()
+          ql.append(catchment_qlats_transposed)
+
+    qlats_df = pd.concat(ql)
```

Comment:
Use `pd.concat(ql, axis='columns').T` and leave out the transposing above.

Reply:
Thanks. Changing this.

Reply:
Note that last few lines of the loop would change to look like
```python
    ...
    catchment_qlats = catchment_qlats.iloc[:, 1]
    ql.append(catchment_qlats)
```